### PR TITLE
UG changes for log command related issues from PED

### LINKED ISSUE #186: [PE-D][Tester D] Missing log details when log is added to the app
1. Add a new log using "log -id 1 -hours 2 -content English Comprehension -style Visual -notes Great improvement! "

Expectation: 
We expect to see a change in the description or the UI when a new log is added.

Actual:
This data cannot be viewed anywhere on the app, it also doesn't show up under the person that the log is added to.

This can be confusing for the user and also the UG does not explain where this information will be displayed.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: shavonneg/ped#13

### PR DESCRIPTION
**Pull request by 0nesheep**
Closes #186 #177 #172 #167 #138

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -237,11 +242,18 @@ Examples:
...
-This will log the lessons of a student. The time field of the log entry will be the system time when the log was added.
-Format `log -id {ID} -hours {HOURS} -content {CONTENT} -style {LEARNING STYLE} -notes {NOTES}
+This will add a log to the lessons of a student. The time field of the log entry will be the *system time* when the log was added.
+Format: `log -id {ID} -hours {HOURS} -content {CONTENT} -style {LEARNING STYLE} -notes {NOTES}`
+<box type="info" seamless>
+Note: Hours need not be an integer input. Values like `2 hours 45 mins`, `2+ hours` are valid as well.
+</box>
 Examples:
-* `log -id 1 -hours 2 -content English Comprehension -style Visual -notes Great improvement!`. would log a lesson for ID 1 noting that it had great improvement in English Comprehension, it also logs the learning style of the student
+* `log -id 000001 -hours 2 -content English Comprehension -style Visual -notes Great improvement!`. would log a lesson for ID 1 noting that it had great improvement in English Comprehension, it also logs the learning style of the student
```
Comment (MarcusGitty) [inline docs/UserGuide.md:251]: Just a nit, should we put 000001 instead of 1 to match the command ?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -105,9 +109,10 @@ Examples:
 
 ### Editing a student: `edit`
 
-Edit the specific student detail from TuteeTally using the ID.
-
-Parameters: INDEX (must be a positive integer) [-nameNAME] [-phonePHONE] [-emailEMAIL] [-addressADDRESS] [-subjectSUBJECT] [t/TAG]...
```
Comment (MarcusGitty) [inline docs/UserGuide.md:110]: Hmm, did you edit the error msg for invalid edit command to remove the subject yet ? <or is this not allowed, im not sure :/ > 

Regarding issue #177 

image

**Review by MarcusGitty — APPROVED** [review]
LGTM, with another comment

File: docs/UserGuide.md (hunk 2 above)
Comment (0nesheep) [reply docs/UserGuide.md:110]: I will amend this together with another command error prompt issue! See issue #163

File: docs/UserGuide.md (hunk 1 above)
Comment (0nesheep) [reply docs/UserGuide.md:251]: Yup! Nice catch

**Pull request merged**